### DISCUSSION TITLE
Disable FPS, GPU, Mem stats overlay

### DISCUSSION THREAD

**Discussion by zemzelett** [julianxhokaxhiu/FFNx#678] [Q&A] [answered]
Hi everyone,

is there a possibility to disable the following overlay:

<img width="241" alt="Screenshot 2024-03-23 at 16 14 02" src="https://github.com/julianxhokaxhiu/FFNx/assets/11561719/eb98c312-63cc-4f60-aac9-5b1f3b37f43e">

This was present by default right after installing FFNx.

I have looked into the `FFNx.toml` in order to disable it but can't seem to find an option in there to do so.

**Comment by eve-atum** [comment]
These are all about 3/4 of the way down in the debugging options group.  That said, your overlay doesn't look like what I'm used to... are you using a Linux variant of some kind?

show_fps = false
show_renderer_backend = false
show_stats = false

**Reply by zemzelett** [reply]
I set all of these options to `false` already. Should have mentioned that, sorry.

I run it on MacOS via Wine.

**Comment by julianxhokaxhiu** [comment]
That's not coming from FFNx but from another software you might be using in your system.

**Reply by zemzelett** [reply]
Thank you. Good to know, so i know i'll have to investigate elsewhere.

**Comment by zemzelett** [comment, marked as answer]
The overlay was coming from [Whisky's](https://github.com/Whisky-App/Whisky) Metal HUD, which i had accidentally enabled.

Turn it off as shown below (and restart the application):

<img width="385" alt="Screenshot 2024-03-23 at 18 33 47" src="https://github.com/julianxhokaxhiu/FFNx/assets/11561719/2fec8388-8f2e-44dc-ace0-5c692694a0ab">
